docs/ZMI.rst

```diff
@@ -10,12 +10,12 @@ browser.
 Bootstrap ZMI
 -------------
 
-Since Zope 4.0b4 the ZMI is styled using Twitter bootstrap. The previously used
+Since Zope 4.0b4 the ZMI is styled using Twitter Bootstrap. The previously used
```

Comment:
Bootstrap is not called "Twitter Bootstrap" since quite a few years already.

https://www.quora.com/Bootstrap-is-no-longer-referred-to-as-Twitter-Bootstrap-Has-Twitter-cut-off-support-for-Bootstrap